detailed_docs/SDL-HMI_API/OnServiceUpdate/OnServiceUpdate_TRS.md

```diff
@@ -0,0 +1,128 @@
+## Functional Requirements
+1.
+In case  
+mobile app successfully registers on SDL  
+and sends StartService request for a service (RPC/Audio/Video)
+
+SDL must  
+send OnServiceUpdate(`serviceType`, `REQUEST_RECEIVED`) notification to HMI
+
+_Note: The only time when SDL would not be able provide the `appID` would be during the first StartService request for the RPC service before RAI was sent._
+
+2.
+In case  
+mobile app requested StartService  
+SDL has valid certificates  
+and the requested service successfully started  
+
+SDL must  
+send OnServiceUpdate(`serviceType`, `REQUEST_ACCEPTED`) notification to HMI
+
+3.
+In case  
+force protection for Audio/Video service is switched OFF  
+and mobile app sends StartService(`serviceType`, encryption = true) request to open protected Video/Audio service  
+and the service is not able to start due to one of the following reasons
+* missing or rejected GetSystemTime_response from HMI
+* invalid mobile certificate
+* invalid SDL certificate 
+
+SDL must  
+send OnServiceUpdate (`serviceType`, `REQUEST_ACCEPTED`, `PROTECTION_DISABLED`) to HMI
+send StartServiceACK(`serviceType`, encryption = false) to the app
+
+4.  
+In case  
+mobile app sends StartService request  
+and SDL doesn't have certificates/has expired certificate/certificate is about to expire in next 24 hours  
+and the requested service is not able to start  
+
+SDL must  
+send OnServiceUpdate(`serviceType`, `REQUEST_REJECTED`)
```

Comment:
@KhrystynaDubovyk I suppose we need to mention the `INVALID_CERT` reason here

Reply:
@dboltovskyi, thank you. added here https://github.com/smartdevicelink/sdl_requirements/pull/123/commits/9b1de67d425ca369927d7095a9cb840cfd9173cb